Jaydeep/Assignment1/Q6.py

```diff
@@ -0,0 +1,58 @@
+def readdat(filename):
```

Comment:
I knew you will copy paste! :P But I asked you to add them as attributes to the mesh class you created above! which was not there in the code I gave you!

Reply:
Oh okay, I updated it now.